.changeset/heavy-peas-sneeze.md

```diff
@@ -0,0 +1,6 @@
+---
+'@astrojs/svelte': patch
+'@astrojs/vue': patch
```

Comment:
Should this be a major so that the `-beta.0` is applied to a new major?

Also I don't see the peer deps range updated. I was checking locally with `pnpm changeset version`